src/NHSD.BuyingCatalogue.Identity.UserDatabase/Tables/Organisations.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Organisations]
 (
-	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
-    [Name] VARCHAR(255) NOT NULL, 
-    [OdsCode] VARCHAR(8) NULL, 
-    [LastUpdated] DATETIME2 NOT NULL
-)
+	Id UNIQUEIDENTIFIER NOT NULL,
+    [Name] NVARCHAR(255) NOT NULL,
+    OdsCode NVARCHAR(8) NULL,
+    LastUpdated DATETIME2 NOT NULL DEFAULT GETUTCDATE(),
+    CONSTRAINT PK_Organisations PRIMARY KEY NONCLUSTERED (Id)
```

Comment:
Should this constraint have ASC on the end? Just compared a PK constraint with the Solutions database script.

Reply:
`ASC` is the default here so omitting it just avoids some unnecessary clutter and I believe the key and index definitions follow this convention elsewhere within the same DB project.